exonum/src/proto/schema/exonum/runtime.proto

```diff
@@ -66,34 +66,53 @@ message GenesisConfig {
   repeated InstanceInitParams builtin_instances = 3;
 }
 
-// The kind of ExecutionError.
+// The type of ExecutionError.
+// Note that ErrorKind doesn't communicate the source of the error, for this
+// see the 'runtime_id' and 'call_site' fields of ExecutionError.
```

Comment:
I think both `ErrorKind` and `runtime_id` / `call_site` may be necessary to interpret the error; see the other comment.